### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by fedelocci**
card story EN-620 EN-615

### CODE REVIEW

File: src/test/ui/menu/BrandMenu.test.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  it('shows the Brand logo', () => {
+    expect(component.contains(BRANDLOGO)).toBe(true);
+  });
+
+  it('verify if component has the correct pf class hierarchy', () => {
+    expect(getFirstChild(component, '.BrandMenu').hasClass('navbar navbar-default navbar-pf')).toBe(true);
+    expect(getFirstChild(component, '.navbar.navbar-default.navbar-pf').hasClass('navbar-header')).toBe(true);
+    expect(getFirstChild(component, '.navbar-header').hasClass('navbar-brand')).toBe(true);
+    // expect(getFirstChild(component, '.navbar-header').hasClass('navbar-brand')).toBe(true);
```
Comment (alessioscalici) [inline src/test/ui/menu/BrandMenu.test.js]: This comment does not seem so useful :thinking:

File: src/ui/menu/AdminAppSwitch.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```
Comment (alessioscalici) [inline src/ui/menu/AdminAppSwitch.js]: This file doesn't belong to this activity. Please remove it

File: src/ui/menu/BrandMenu.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
```
Comment (alessioscalici) [inline src/ui/menu/BrandMenu.js]: This file does not belong to this activity. Please remove

File: src/ui/menu/HelpMenu.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```
Comment (alessioscalici) [inline src/ui/menu/HelpMenu.js]: This file does not belong to this activity. Please remove

File: src/ui/menu/ProjectLink.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```
Comment (alessioscalici) [inline src/ui/menu/ProjectLink.js]: This file does not belong to this activity. Please remove

File: src/ui/menu/UserDropdown.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```
Comment (alessioscalici) [inline src/ui/menu/UserDropdown.js]: This file does not belong to this activity. Please remove

**Comment by fedelocci** [comment]
create new PR EN-620

**Pull request closed without merging**